packages/cactus-plugin-satp-hermes/src/main/typescript/core/stage-handlers/client/stage1-client-handler.ts

```diff
@@ -137,75 +156,85 @@ export class Stage1ClientHandler {
     // transferProposalRequestMessage.multipleClaimsAllowed = sessionData.multipleClaimsAllowed;
     // transferProposalRequestMessage.multipleCancelsAllowed = sessionData.multipleCancelsAllowed;
 
-    const messageSignature = PluginSATPGateway.bufArray2HexStr(
-      gateway.sign(JSON.stringify(transferProposalRequestMessage)),
+    const messageSignature = bufArray2HexStr(
+      sign(
+        gateway.gatewaySigner,
+        JSON.stringify(transferProposalRequestMessage),
+      ),
     );
 
     transferProposalRequestMessage.common.signature = messageSignature;
 
     sessionData.signatures = new MessageStagesSignatures();
     sessionData.signatures.stage1 = new Stage1Signatures();
-    sessionData.signatures.stage1.transferCommenceRequestMessageClientSignature = messageSignature;
-    
+    sessionData.signatures.stage1.transferCommenceRequestMessageClientSignature =
+      messageSignature;
+
     sessionData.hashes = new MessageStagesHashes();
     sessionData.hashes.stage1 = new Stage1Hashes();
-    
+
     sessionData.hashes.stage1.transferCommenceRequestMessageHash = SHA256(
       JSON.stringify(transferProposalRequestMessage),
     ).toString();
 
     //gateway.sessions.set(sessionID, sessionData);
-    
-    await gateway.storeLog({ //todo
-      sessionID: sessionID,
-      type: "init",
-      operation: "validate",
-      data: JSON.stringify(sessionData),
-    });
+
+    // await gateway.storeLog({
+    //   //todo
```

Comment:
why is this code deleted? we need this functionality

Reply:
Same reason from above